src/density-qvm.lisp

```diff
@@ -368,7 +368,7 @@ EXCITED-PROBABILITY should be the probability that QUBIT measured to |1>, regard
     ;; Return the qvm.
     (values qvm cbit)))
 
-(defmethod transition ((qvm density-qvm) (instr quil:measure))d
+(defmethod transition ((qvm density-qvm) (instr quil:measure))
   (let ((ret-qvm (call-next-method qvm instr)))
```

Comment:
are there other CALL-NEXT-METHODs that are modifying the PC correctly?

Reply:
Can we get rid of this `ret-qvm` stuff, since we're mutating the `qvm` object anyway.

Reply:
reasonable to me